# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/browser/browser_103_preload.js
# Commit: a6a5172b9746
# Full Hash: a6a5172b9746011a379a5f0b603beb3f3d711315
# Author: Manuel Bucher <manuel@mozilla.com>
# Date: 2022-12-03 09:24:59
# Regressor Bug: 1771867
# File Overlap Count: 1
# Description:
#   Bug 1771867 - Early Hints Phase 2 - Part 9: Don't cancel redirects to insecure http early hint preloads r=necko-reviewers,valentin
#   
#   Previously those got cancelled to discourage http usage in new components.
#   Preloading them now to make this preloader behave the same way as the
#   preloader in the content process.
# ==============================================================================

diff -r 8575cd92e1d8 -r a6a5172b9746 netwerk/test/browser/browser_103_preload.js
--- a/netwerk/test/browser/browser_103_preload.js	Fri Dec 02 16:15:56 2022 +0000
+++ b/netwerk/test/browser/browser_103_preload.js	Fri Dec 02 16:15:56 2022 +0000
@@ -145,7 +145,7 @@
     "test_103_preload_insecure_cor",
     "https://example.com",
     "http://mochi.test:8888/browser/netwerk/test/browser/early_hint_pixel.sjs",
-    { hinted: 0, normal: 1 }
+    { hinted: 1, normal: 0 }
   );
 });
 